src/jit/target.h

```diff
@@ -246,6 +246,7 @@ typedef unsigned char   regNumberSmall;
   #define FEATURE_MULTIREG_ARGS_OR_RET  1  // Support for passing and/or returning single values in more than one register
   #define FEATURE_MULTIREG_ARGS         0  // Support for passing a single argument in more than one register  
   #define FEATURE_MULTIREG_RET          1  // Support for returning a single value in more than one register
+  #define MAX_PASS_SINGLEREG_BYTES      8  // Maximum size of a struct passed in a single register (double).
```

Comment:
Shouldn't this be 16 (for SIMD types)?

Reply:
Oh, I see. This is x86.

Reply:
Yes, someday I hope we can split up target.h so that it's not so easy to get confused about which target you're actually looking at!